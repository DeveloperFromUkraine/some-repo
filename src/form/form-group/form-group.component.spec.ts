import { ComponentFixture, TestBed } from '@angular/core/testing';
import { FormGroupComponent } from './index';
import { ComponentTest } from '../../../test/test-bed/component';

describe('Form Group', () => {
  let fixture: ComponentFixture<FormGroupComponent>;

  beforeEach(async () => {
    ComponentTest.createTestBed([], [FormGroupComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(FormGroupComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
