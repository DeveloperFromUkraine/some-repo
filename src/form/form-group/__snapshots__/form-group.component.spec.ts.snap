// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Group should match snapshot 1`] = `
<ign-form-group>
  
</ign-form-group>
`;
